src/zimscraperlib/zim/creator.py

```diff
@@ -146,7 +150,30 @@ def config_indexing(
         self.__indexing_configured = True
         return self
 
+    def _get_illustration_metadata(
+        self,
+        name: str,
+        value: bytes | str | datetime.datetime | datetime.date | Iterable[str],
+    ) -> str | None:
+        """Return image format for debug logging of illustration metadata"""
+        if not name.startswith("Illustration_"):
```

Comment:
Makes little sense to pass every metadata into this and return if name is not illus-like. Why not having this check prior to calling it?